### PR TITLE
contact form link and 2000.html

### PR DESCRIPTION
**Pull request by lkleinberg**
See comment re. 2000.html - I think I fixed the problem with all the nested div tags, but we might have to see how it looks in production...

### CODE REVIEW

**Comment by stopfstedt** [comment]
@Trott please review and merge if OK. Thanks.

**Comment by Trott** [comment]
This looks no good to me. There are now three more `</div>` elements than `<div>` elements.

**Comment by lkleinberg** [comment]
Ah - thanks - I think that problem was already there (and was worse), as I actually closed some div's that needed closing. I will take another look.

**Comment by stopfstedt** [comment]
here an example of a correctly formed date content block

``` html
<div class="dates">
    <div class="dates_text">
        <strong>2010</strong>
    </div>
    <div class="dates_body">David Julius, PhD receives the 2010 Shaw Prize in Life Sciences and Medicine for his groundbreaking work on the sensation of touch.
    </div>
</div>
```

**Comment by Trott** [comment]
I tweaked the page and submitted a pull request to your branch, Leslie. If you want to see what the site will look like with my changes, go to http://trott.github.io/ucsf-history-website/

**Comment by Trott** [comment]
Annnd, I goobered it....
Let me try again.

**Comment by Trott** [comment]
OK, all set.
